plugins/logitech-rallysystem/fu-logitech-rallysystem-audio-device.c

```diff
@@ -194,6 +194,17 @@ fu_logitech_rallysystem_audio_device_set_progress(FuDevice *self, FuProgress *pr
 static gchar *
 fu_logitech_rallysystem_audio_device_convert_version(FuDevice *device, guint64 version_raw)
 {
+	guint8 major = 0;
+	guint8 minor = 0;
+	guint8 build = 0;
+	/*
+	 * device reports system version in 3 bytes: major.minor.build
+	 * convert major.minor.build -> major.minor.0.build
+	 */
+	major = (version_raw >> 16) & 0xFF;
+	minor = (version_raw >> 8) & 0xFF;
+	build = (version_raw >> 0) & 0xFF;
+	version_raw = ((major << 24) | (minor << 16) | 0 | (build << 0));
```

Comment:
I think coverity will hate the | 0, can you remove that and the outer brackets please

Reply:
Done.